### ISSUE TITLE
Set custom trace level in options

### ISSUE THREAD

**Issue by pkozlovskiy** [infinum/floggy#26]
I have abstract layer of logger service and use loggy like:

```
class LoggyLogger extends AppLoggerService with UiLoggy {
  @override
  void debug(dynamic message, [Object? error, StackTrace? stackTrace]) {
    loggy.debug(message, error, stackTrace);
  }
```

But in this way i can`t use includeCallerInfo (

**Comment by itsJoKr** [comment]
Hey @pkozlovskiy, can you explain what do you mean by `includeCallerInfo`? The only thing I've found is that other logging package simple_logger has that kind of parameter, which will show caller that's clickable. 

@lukaknezic can we replicate this in PrettyDeveloperPrinter.

**Comment by pkozlovskiy** [comment]
You right this is about show clickable caller and in this place
https://github.com/infinum/floggy/blob/19995ab32de06067c9d5624b5cbc3c5cf5321e78/loggy/lib/src/loggy.dart#L249
Level is hardcoded as 
`    const _level = 3;`
and that why i always get line of my wrapper caller...

**Comment by lukaknezic** [comment]
Hey @pkozlovskiy 

We have `includeCallerInfo` but they are in `logOptions`. 

You should set `logOptions` in `Loggy.initLoggy` then it should work for all logs. Can you try including it in `initLoggy` that should fix your problem 🙂

**Comment by lukaknezic** [comment]
Ohh okay I understand your problem now. Hmmm we will have to add something that will allow you to change that.

**Comment by itsJoKr** [comment]
@pkozlovskiy Can you check our PR. Will something like this solve your problem: https://github.com/infinum/floggy/pull/27#issuecomment-876226897

**Comment by pkozlovskiy** [comment]
Thanks! This work for me!

**Comment by lukaknezic** [comment]
Fixed in: `loggy: ^2.0.0+1`
